server/src/main/scala/cromwell/CommandLineParser.scala

```diff
@@ -79,7 +81,11 @@ class CommandLineParser extends scopt.OptionParser[CommandLineArguments]("java -
         opt[String]('m', "metadata-output").text(
           "An optional JSON file path to output metadata.").
           action((s, c) =>
-            c.copy(metadataOutput = Option(DefaultPathBuilder.get(s))))
+            c.copy(metadataOutput = Option(DefaultPathBuilder.get(s)))),
+        opt[String]('w', "workflow-output").text(
```

Comment:
TOL: `-w` felt a little clunky but since `-o` is already taken,  guess it's ok...

Reply:
Why not `-j` and `--json-output`? This follows the name in the to be spec (JSON output format) as well as being more specific than "workflow output"